ulimit

```diff
@@ -0,0 +1,5 @@
+# Show help
```

Comment:
>  \# Show help

My read of the `man` page is that `ulimit -a` reports "current limits" rather than "help". Do you agree with that statement, or is this an inconsistency between platforms/shells?